response.go

```diff
@@ -35,6 +36,15 @@ type Response struct {
 func (or *Response) readSGMLHeaders(r *bufio.Reader) error {
 	var seenHeader, seenVersion bool = false, false
 	for {
+		// sometimes there's no newline between the SGML headers and the XML payload
```

Comment:
To make sure everyone's clear that we're not dealing with valid OFX here, and to avoid claiming that the payload is XML, can we update this comment to be something like:
```suggestion
		// Some financial institutions do not properly leave an empty line after the last header. Avoid attempting to read another header in that case.
```